apps/bmd/src/components/PrecinctTallyReport.tsx

```diff
@@ -110,7 +111,10 @@ const PrecinctTallyReport = ({
                         <TD narrow textAlign="right">
                           {isContestInPrecinct
                             ? numberWithCommas(
-                                (tally[contestIndex] as MsEitherNeitherTally)
+                                // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
I had to add this or the type assertion kept getting removed, this code will be replaced in a future PR that will replace this entire section with the UI components for tally reports from election-manager.

Reply:
I think you're seeing this problem locally but it's not actually a problem. Perhaps you need to `pnpm install` and restart VS Code? Run `pnpm build:watch` in `bmd`?

https://app.circleci.com/pipelines/github/votingworks/vxsuite/1927/workflows/9b9a442e-7984-45a5-ace4-47dd1b49f546/jobs/25149?invite=true#step-107-14

Reply:
Here's a way to fix these issues: https://github.com/votingworks/vxsuite/commit/65863dd6981640e3faf344b968354ac01ea5a1e7.

Reply:
Ah had to nuke node modules and re-install to fix it